### DISCUSSION TITLE
Monorepos sử dụng chung versions

### DISCUSSION THREAD

**Discussion by VuiLenDi** [nashtech-garage/nodejs-msa#142] [Ideas]
Suy nghĩ dùng chung versions (nest.js)

**Comment by tuonglevan** [comment]
a @VuiLenDi, We need to use the same version of NestJS throughout a monorepo provide several advantages, such as:
- Consistency
- Simplified Maintenance
- Shared Documentation
- Unified Test Environment
- Dependency Management
- Compatibility
- Code Reusability
- Easy to maintain.

**Comment by tuonglevan** [comment]
Structure backend for NestJS 
<img width="722" alt="image" src="https://github.com/user-attachments/assets/ea835937-1c19-41f9-986b-c066c4b62e9b">
